src/admin/datatypes/index.md

```diff
@@ -3,6 +3,5 @@
 Hub page for [administering](/src/admin/index.md) Galaxy data types.
 
 * [Admin/ReferenceMAFs](/src/admin/reference-mafs/index.md)
-* [Adding Completely new Datatypes](/src/admin/datatypes/Adding Complete Datatypes/index.md)
```

Comment:
why was this dropped? it lives at https://galaxyproject.org/admin/datatypes/adding-complete-datatypes/